.ci/config/stage-types.yaml

```diff
@@ -55,6 +55,8 @@ instance-migration:
   stash:
     runtimeStash: true
     qaStash: true
+  labels:
+    - 'all-db'
```

Comment:
**Reminder:** Revert the changes in this file
**Why**: It's used only for testing purposes to test the migration tests against **db2** & **mssql**

Reply:
Done